components/BackButton.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import Head from 'next/head';
+
+const BackButton = () => {
+console.log('BackButton being rendered')
+  return (
+    <div>
+      <main>
```

Comment:
<main> is a container for the main section of the entire app. This should not be used here.